config/releases.exs

```diff
@@ -40,7 +40,7 @@ config :amqp,
     events: [connection: :rabbitmq_conn]
   ]
 
-config :warehouse, Warehouse.AssemblyServiceClient,
+config :warehouse, Warehouse.Clients.Assembly.Connection,
```

Comment:
Clients to interface with other systems are now isolated on it's own directory and use an OTP behaviour so they can be easily mocked and tested.